index.js

```diff
@@ -410,6 +410,71 @@ const getPhpVersionVulnerabilitiesOfBitbucketRepo = async (
   return vulnerabilities.filter(vulnerability => vulnerability !== undefined)
 }
 
+/**
+ * Report any use of an End-Of-Life'd (EOL'd) PHP version in the specified
+ * Bitbucket repos.
+ *
+ * @param {string} bitbucketUsername - A Bitbucket username
+ * @param {string} bitbucketAppPassword - A Bitbucket app password
+ * @param {string[]} repos - The list of Bitbucket repos (e.g. ["owner/library"])
+ * @param {string} versionsCsvUrl - (Optional:) The URL where we can retrieve a
+ *     CSV file of the PHP versions used by various Docker images.
+ * @returns {Promise<{repo, detailsUrl, ecosystem, identifiers, inspectedOn, packageName, severity, summary, vulnerableVersionRange}[]>}
+ */
+const getPhpVersionVulnerabilitiesOfBitbucketRepos = async (
+  bitbucketUsername,
+  bitbucketAppPassword,
+  repos,
+  versionsCsvUrl = ''
+) => {
+  const results = []
+  for (let i = 0; i < repos.length; i++) {
```

Comment:
Just curious, any reason you prefer or needed a for loop rather than a for of loop?

Reply:
Nope, good point. I just have trouble remembering the nuances of JavaScripts various looping mechanisms. It looks like for/of would work great here.

Reply:
Okay, done.